### PR TITLE
docs: fix removeAdditional comment

### PR DESCRIPTION
**Pull request by G0maa**
Small fix/correction for https://github.com/fastify/website/issues/140

#### Checklist

- [ ] run `npm run test` and `npm run benchmark`
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: docs/Reference/Validation-and-Serialization.md [hunk 1]
```diff
@@ -397,7 +397,7 @@ configuration](https://github.com/fastify/ajv-compiler#ajv-configuration) is:
 {
   coerceTypes: 'array', // change data type of data to match type keyword
   useDefaults: true, // replace missing properties and items with the values from corresponding default keyword
-  removeAdditional: true, // remove additional properties
+  removeAdditional: true, // To remove additional properites refer to: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties
```
Comment (Uzlopak) [inline docs/Reference/Validation-and-Serialization.md:400]:
```suggestion
  removeAdditional: true, // remove additional properties if additinalProperties is set to false, see also: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties
```

**Review by Uzlopak — CHANGES_REQUESTED** [review]
see comment

File: docs/Reference/Validation-and-Serialization.md (hunk 1 above)
Comment (G0maa) [reply docs/Reference/Validation-and-Serialization.md:400]: If we're going the detailing path, I'd suggest something like `// remove additional properties if additinalProperties is set to false in JSON Schema, see also: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties` 😁

**Review by Fdawgs — CHANGES_REQUESTED** [review]
Will provide proper review when I get a spare second, can be rewritten to be more concise.

File: docs/Reference/Validation-and-Serialization.md [hunk 2]
```diff
@@ -397,7 +397,7 @@ configuration](https://github.com/fastify/ajv-compiler#ajv-configuration) is:
 {
   coerceTypes: 'array', // change data type of data to match type keyword
   useDefaults: true, // replace missing properties and items with the values from corresponding default keyword
-  removeAdditional: true, // remove additional properties
+  removeAdditional: true, // remove additional properties if additinalProperties is set to false, see also: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties
```
Comment (sceccotti89) [inline docs/Reference/Validation-and-Serialization.md:400]: Typo here: `additinalProperties`

Comment (G0maa) [reply docs/Reference/Validation-and-Serialization.md]: Fixed.

File: docs/Reference/Validation-and-Serialization.md [hunk 3]
```diff
@@ -397,7 +397,7 @@ configuration](https://github.com/fastify/ajv-compiler#ajv-configuration) is:
 {
   coerceTypes: 'array', // change data type of data to match type keyword
   useDefaults: true, // replace missing properties and items with the values from corresponding default keyword
-  removeAdditional: true, // remove additional properties
+  removeAdditional: true, // remove additional properties if additionalProperties is set to false, see also: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties
```
Comment (Fdawgs) [inline docs/Reference/Validation-and-Serialization.md:400]:
```suggestion
  removeAdditional: true, // remove additional properties if additionalProperties is set to false, see: https://ajv.js.org/guide/modifying-data.html#removing-additional-properties
```

**Review by Uzlopak — APPROVED** [review]
LGTM

@Fdawgs 
Please approve and merge

**Review by Fdawgs — APPROVED** [review]

**Pull request merged**